server/src/main/java/io/spine/server/procman/DefaultProcessManagerRepository.java

```diff
@@ -0,0 +1,30 @@
+package io.spine.server.procman;
+
+import com.google.protobuf.Message;
+import io.spine.server.procman.model.ProcessManagerClass;
+
+import static io.spine.server.procman.model.ProcessManagerClass.asProcessManagerClass;
+
+/**
+ * Default implementation of {@code ProcessManagerRepository}.
+ */
+public final class DefaultProcessManagerRepository<I,
+                                                   P extends ProcessManager<I, S, ?>,
+                                                   S extends Message>
+    extends ProcessManagerRepository<I, P, S> {
+
+    private final ProcessManagerClass<P> modelClass;
+
+    /**
+     * Creates a new repository managing process managers of the passed class.
+     */
+    public DefaultProcessManagerRepository(Class<P> cls) {
```

Comment:
If we delegate the default repository instantiation to the `EntityClass`es, this constructor should accept a `ProcessManagerClass`, not just a `Class`.

Reply:
As discussed vocally, we deliberately violate OOP in this case in favor of concise syntax for creating default repositories. This way it would be convenient for the user of the framework, and the code reads much better this way.